examples/graphql-config/package.json

```diff
@@ -13,7 +13,7 @@
   },
   "devDependencies": {
     "@eslint/js": "9.15.0",
-    "@graphql-eslint/eslint-plugin": "workspace:*",
+    "@graphql-eslint/eslint-plugin": "4.0.0-alpha.16",
```

Comment:
Was this intentional? You'll have to maintain the version now.

Reply:
yeah this is intentional, otherwise, we need to modify manually version in Stackblitz, renovate will create a PRs with new versions when they will out